CHANGELOG.rst

```diff
@@ -20,6 +20,11 @@ Changed
   official instructions at https://docs.mongodb.com/v3.4/release-notes/3.4-upgrade-standalone/.
   (improvement)
 
+Deprecated
+~~~~~~~~~~
```

Comment:
I think this section heading should be "Removed"
We Deprecated it in an earlier version, now it should be Removed.